packages/badge/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Products } from '@hashicorp/platform-product-meta'
+import classnames from 'classnames'
+import s from './style.module.css'
+
+interface BadgeProps {
+  children: string
```

Comment:
I'm curious if you considered using a prop for the text value instead, like `<Badge title="" />`?

Reply:
I wanted to propose we use children because it naturally follows how you'd use an html element. For example you want to place text inside a paragraph `<p>Test</p>` vs `<p title="Test" />`. I've found it to be confusing the use of text or title between our components. [Button](https://react-components.vercel.app/components/button) expects `title`, where [CallToAction](https://react-components.vercel.app/components/calltoaction) links expect `text`.

Reply:
+1 - That was what I had in t[his initial iteration on a project](https://hashicorp-www-next-git-jmhomepage-promo-section-hashicorp.vercel.app/swingset/components/badge), I wondered if there was a strong reason _for_ using `children`

Reply:
+100! The "pure HTML" feel of wrapping an element's content is very intuitive, and as a bonus it promotes composition. I dig this!